modules/firelens_log_config/main.tf

```diff
@@ -0,0 +1,152 @@
+# This is just a module to provide a template for creating a log configuration for an ECS task definition.
+
+locals {
+  tld = var.region == "eu-central-1" ? "eu" : "com"
+
+  fire_lens_container = <<EOF
+  {
+    "essential": true,
+    "image": "906394416424.dkr.ecr.${var.region}.amazonaws.com/aws-for-fluent-bit:stable",
+    "name": "log_router",
+    "cpu": 0,
+    "user": "0",
+    "environment": [],
+    "volumesFrom": [],
+    "portMappings": [],
+    "mountPoints": [],
+    "firelensConfiguration": {
+      "type": "fluentbit",
+      "options": {
+        "enable-ecs-log-metadata": "true"
+      }
+    },
+    "memoryReservation": 50,
+    "dockerLabels": {
+      "com.datadoghq.tags.env": "${var.environment_name}",
+      "com.datadoghq.tags.service": "${var.app_name}",
+      "com.datadoghq.tags.version": "${var.datadog_image}"
```

Comment:
Ahhh I was wondering why this was showing the full image name. We'll only need the tag version..

Reply:
Tags: [here](https://github.com/smartrent/nerves-hub-terraform/pull/30/files#diff-f275be9fd58c428c299725380fb9b144a187f2f678014da65f86108b50da97e8R27), [here](https://github.com/smartrent/nerves-hub-terraform/pull/30/files#diff-f275be9fd58c428c299725380fb9b144a187f2f678014da65f86108b50da97e8R112), [here](https://github.com/smartrent/nerves-hub-terraform/pull/30/files#diff-f275be9fd58c428c299725380fb9b144a187f2f678014da65f86108b50da97e8R128), [here](https://github.com/smartrent/nerves-hub-terraform/pull/30/files#diff-f275be9fd58c428c299725380fb9b144a187f2f678014da65f86108b50da97e8R144), [here](https://github.com/smartrent/nerves-hub-terraform/pull/30/files#diff-f275be9fd58c428c299725380fb9b144a187f2f678014da65f86108b50da97e8R158), [here](https://github.com/smartrent/nerves-hub-terraform/pull/30/files#diff-f275be9fd58c428c299725380fb9b144a187f2f678014da65f86108b50da97e8R172), [here](https://github.com/smartrent/nerves-hub-terraform/pull/30/files#diff-f275be9fd58c428c299725380fb9b144a187f2f678014da65f86108b50da97e8R186), and [variable is here](https://github.com/smartrent/nerves-hub-terraform/pull/30/files#diff-c721832437594336b1ddcbaf06a2ea047b6eff6dc5b37b9c972d327e703adfceR30)